src/less/snackbar-dialog/snackbar-dialog.less

```diff
@@ -3,17 +3,19 @@
 @import "../mixins/private/token-mixins.less";
 
 .snackbar-dialog {
-    .background-color-token(snackbar-dialog-background-color, color-background-secondary);
-    bottom: 0;
+    .background-color-token(snackbar-dialog-background-color, color-background-inverse);
+    .border-radius-token(snackbar-dialog-border-radius, border-radius-100);
+    bottom: 40px;
     box-shadow: 0 0 3px rgba(0, 0, 0, 0.28);
-    .color-token(snackbar-dialog-foreground-color, color-foreground-primary);
-    left: 0;
+    .color-token(snackbar-dialog-foreground-color, color-foreground-on-inverse);
+    left: 8px;
+    margin: auto;
     max-height: 40vh;
-    min-width: 320px;
+    max-width: 448px;
```

Comment:
I'm not seeing any `min-width` or `width` (so I would expect the shorter content snackbars to collapse), but it's somehow staying at a fixed width. If there is any scenario under which is gets collapsed, we might want to add `min-wdith` since the specs show all the snackbars at the same width.

Reply:
The only concern that I have with setting `min-width` is that I think the dialog will overflow for screens with a width of less than `(320+8)px`. A quick web search tells me that there is still a non-negligible population of web users who have a screen width of `320px`. If you are worried about min-width, I think an alternative solution to centering is the following:

```css
.snackbar-dialog {
  left: 0;
  right: 0;
  width: calc(100vw - 16px);
  max-width: 448px;
}
```

I avoided this because it includes a `calc`, but I can switch if you'd like. That said, I haven't found any cases where the dialog collapses horizontally.

Reply:
If you haven't seen any scenarios under which the dialog gets smaller (small enough just to snug the content), then it's probably not something we should be concerned about, at least for now.

P.S. If we were to set the `min-width`, we could simply use margins (I typically prefer that anyway) to create the separation from the viewport as well as use media queries to define the mins. But, yea, we don't have to worry about that for now.